### PR TITLE
Updated: Typos in a11y-basics file

### PR DESCRIPTION
**Pull request by Cyebukayire**
## Description of Problem
Corrected Grammer, Punctuations and Typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,13 +17,17 @@ git clone git@github.com:vuejs/docs.git
 2. Install dependencies
 
 ```bash
-yarn # or npm install
```
Comment (dammy001) [inline README.md:20]: I don't think you have to change this

Comment (Cyebukayire) [reply README.md:20]: Just thought of making it clear
but it's okay

**Review by Cyebukayire — COMMENTED** [review]
I've corrected it now.
I mean the unnecessary changes that I made in the README file.
So, is it a good now?

**Comment by dammy001** [comment]
Yeah... @skirtles-code

**Comment by Cyebukayire** [comment]
when will the changes be merged?

File: src/cookbook/editable-svg-icons.md [hunk 2]
```diff
@@ -164,4 +164,4 @@ Other tooling to help manage SVG icons includes:
 - [svg-sprite-loader](https://github.com/kisenka/svg-sprite-loader)
 - [svgo-loader](https://github.com/rpominov/svgo-loader)
 
-These tools bundle SVGs at compile time, but make them a little harder to edit during runtime, because `<use>` tags can have strange cross-browser issues when doing anything more complex. They also leave you with two nested `viewBox` properties and thus two coordinate systems. This makes the implementation a little more complex.
+These tools bundle SVGs at compile-time, but make them a little harder to edit during runtime, because `<use>` tags can have strange cross-browser issues when doing anything more complex. They also leave you with two nested `viewBox` properties and thus two coordinate systems. This makes the implementation a little more complex.
```
Comment (NataliaTepluhina) [inline src/cookbook/editable-svg-icons.md:167]: **suggestion**: I disagree with this change: I could easily replace "compile time" with "time of compilation". Yes, Grammarly suggests the hyphen but we shouldn't follow Grammarly suggestions blindly.

**Review by NataliaTepluhina — CHANGES_REQUESTED** [review]

**Comment by NataliaTepluhina** [comment]
@Cyebukayire thank you for your contribution! I've left a suggestion regarding `compile time` change, the rest of the fixes look good to me. I'll merge the PR as soon as you implement the suggestion.

File: src/cookbook/editable-svg-icons.md (hunk 2 above)
Comment (Cyebukayire) [reply src/cookbook/editable-svg-icons.md]: Thank you so much @NataliaTepluhina for the suggestion, Yeah it makes sense
I've changed "compile time" to "time of compilation"

**Comment by NataliaTepluhina** [comment]
Thank you for the fix @Cyebukayire! Happy to merge the PR now 🎉

**Review by NataliaTepluhina — APPROVED** [review]

**Pull request merged**